packages/KInput/KInput.vue

```diff
@@ -49,11 +65,25 @@ export default {
     help: {
       type: String,
       default: ''
+    },
+    size: {
+      type: String,
+      default: 'medium'
+    },
+    /**
+     * Debug mode - for testing only, strips out generated ids
+     */
+    debugMode: {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏is there a better name than debug mode? Seems like something that should get turned off, maybe `testingMode`? or `staticIds`?

Reply:
Changed to `testMode`.